l10n_it_website_sale_corrispettivi/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Italian Localization - E-commerce: ricevute',
+    'summary': 'L\'utente e-commerce può scegliere tra fattura e ricevuta',
```

Comment:
Vedo che nel titolo è stato usato il carattere : .

Se è un caso d'uso diverso rispetto a quello indicato nelle linee guida sarebbe da definire (e poi documentare).
https://github.com/odoo-italia/documentazione/blob/master/sviluppo/guidelines.rst

In caso contrario sarebbe da uniformare.
Inoltre, come da nuove linee guida, potremmo già iniziare a indicare ITA al posto di "Italian Localization".

Forse come sommario/riepilogo andrebbe meglio una descrizione di cosa fa il modulo al posto di un'affermazione su cosa può fare l'utente:
Es:
"Aggiunge la ricevuta come opzione per l'utente e-commerce.

Per il resto direi che è ok. :+1:

Reply:
@primes2h giusto, grazie, modificato

Reply:
@eLBati 
Mi sembra manchi la rigenerazione del README.

Reply:
@primes2h vero, fatto. (la avrebbe comunque lanciata l'OCA bot dopo il merge, ma se possibile è meglio lanciarla prima)

Reply:
:+1: